store/server/features/okrplanning/monitoring-evaluation/reprimand-log/queries.ts

```
@@ -1,12 +1,10 @@
 import { OKR_AND_PLANNING_URL } from '@/utils/constants';
 import { useQuery } from 'react-query';
-import { useAuthenticationStore } from '@/store/uistate/features/authentication';
-import axios from 'axios';
+import { crudRequest } from '@/utils/crudRequest';
+import { requestHeader } from '@/helpers/requestHeader';
 import { ReprimandLog } from '@/store/uistate/features/okrplanning/monitoring-evaluation/reprimand-log/interface';
 import { getCurrentToken } from '@/utils/getCurrentToken';
```

ESLint

[error] 'getCurrentToken' is defined but never used.

(@typescript-eslint/no-unused-vars)